### DISCUSSION TITLE
How are the stable and lazer beatmaps organized?

### DISCUSSION THREAD

**Discussion by LordXerus** [ppy/osu#20105] [Q&A] [answered]
I need to convert my lazer beatmaps to stable for McOsu. As far as I know, there isn't a reverse import in lazer nor a lazer import in McOsu, so can I have some pointers on how the lazer database is organized?

**Comment by Theighlin** [comment, marked as answer]
Read https://github.com/ppy/osu/wiki/User-file-storage. You can manually export single beatmaps from inside the lazer editor, but an all beatmap export doesn't exist.

**Comment by LordXerus** [comment]
Is there a file I can read to obtain the beatmap list and SHA-256 hashes?

**Reply by peppy** [reply]
client.realm

please don't ask for further details though. use the source code if you must
